make-util.js

```diff
@@ -824,10 +824,11 @@ var getRefs = function () {
 
             var release = parseInt(branch.split('/').pop().substr(1));
 
-            // filter out releases less than 108 and greater than HEAD
+            // filter out releases less than 108 and greater than 116 (because releases/dev15tfsu2rc1 branched from releases/m117)
             if (release < 108 ||
-                release > (info.head.release || 999)) {
+                release > 116) {
```

Comment:
in this branch, the getRefs function is only used in one place now: when building the aggregated zip.

so for this branch we need have a ceiling of 116 when adding older major versions to the aggregated zip.